packages/runtime-dom/src/apiCustomElement.ts

```diff
@@ -217,13 +217,15 @@ export class VueElement extends BaseClass {
     const resolve = (def: InnerComponentDef) => {
       const { props, styles } = def
       const hasOptions = !isArray(props)
```

Comment:
The original bug was failing because this would return true when props was undefined.
```suggestion
      const hasOptions = props && !isArray(props)
```

With this change, the whole loop a bbit further down, with `if (hasOptions)` could be skipped